unity/Hello_World/Assets/MixedRealityToolkit.Generated/CustomProfiles/New MixedRealitySpatialAwarenessMeshObserverProfile.asset

```diff
@@ -0,0 +1,29 @@
+%YAML 1.1
```

Comment:
Why are there 3 versions of this file? Are they all needed?

Reply:
No... was experimenting with different mesh observer profiles. I removed the unused ones.